person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(name, age = 'unknown', parent_permission = 'true')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
- [ ] As mentioned by the previous reviewer and for **[the project's requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md)**, kindly make sure the `initialize()` method also has the `@id` as Constance variable and can be generated randomly(.e.g.: ` @id = Random.rand(1..1000`). Make sure you add it to the other instance vars of the initialize method.